kiva/fonttools/_scan_sys.py

```diff
@@ -109,6 +109,38 @@ def scan_system_fonts(fontpaths=None, fontext="ttf"):
     return [fname for fname in fontfiles if os.path.exists(fname)]
 
 
+def scan_user_fonts(fontpaths=None, fontext="ttf"):
+    """ Search for fonts in the specified font paths.
```

Comment:
It might be useful to specify in the docstring that this returns a list of unique font files for the given path.

Reply:
Done